{"uuid": "6cee1575-f74f-43a1-8878-894c2fee8f91", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "8c37162d-ca7f-4619-971e-8ac5cf47767a", "event": "runner_on_unreachable", "pid": 4682, "created": "2022-04-22T13:05:34.175382", "parent_uuid": "9e73ed77-7fb2-eb75-f688-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "37fc0dcf-9c9b-49b2-ac1c-c3c3b8b68fdb", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-eb75-f688-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-eb75-f688-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-22T13:05:34.148897", "end": "2022-04-22T13:05:34.175300", "duration": 0.026403, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "6cee1575-f74f-43a1-8878-894c2fee8f91"}}